doc/classes/LargeTexture.xml

```diff
@@ -20,21 +20,21 @@
 			<argument index="1" name="texture" type="Texture">
 			</argument>
 			<description>
-				Add another [Texture] to this [LargeTexture], starting on offset "ofs".
+				Add another [Texture] to this [code]LargeTexture[/code], starting on offset "ofs".
```

Comment:
That was a reference to a class

Reply:
Yes but it's referencing itself, we must avoid this, so [code] was used, we talked about that on discord channel. @cbscribe @NathanLovato it's right ?

Reply:
Yes, better avoid linking the class if it's already the current page. It looks like a hyperlink to somewhere you should check, it's just confusing. hence the [code] tags

Reply:
In that case i dont even know why include code tags, if you are not talking about a member or anything code like, you are talking about the object itself.

Reply:
For consistency. We highlight all symbols and values with [code][/code] to make the docs consistent. We've had tons of ``, '', "", [code], [ClassName] etc. to fix from the Godot 2.x classref.
The style helps to spot values while reading the class ref. In this case the fact it's a class, and the current class as it's not a hyperlink. And symbols are identifiers, a <pre> or <code> tag works for them.